srcpkgs/python3-pykwalify/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-pykwalify'
+pkgname=python3-pykwalify
+version=1.6.1
+revision=1
+wrksrc=pykwalify-${version}
+build_style=python3-module
+pycompile_module="pykwalify"
+hostmakedepends="python3-setuptools python3-devel"
+makedepends="${hostmakedepends/python3-setuptools/}"
+depends="python3-yaml python3-docopt python3-dateutil"
+short_desc="Python YAML/JSON schema validation library"
+maintainer="Renato Aguiar <renato@renag.me>"
+license="custom"
```

Comment:
Looks like MIT

Reply:
I also thought that, but it has an extra statement ("The Software shall be used for Good, not Evil.") that seems to make it nonfree: https://www.gnu.org/licenses/license-list.html#JSON

Reply:
I think you're right -- and that is the JSON license. In that case it should be JSON https://spdx.org/licenses/JSON.html